src/palette/internal/palettecreator.cpp

```diff
@@ -1603,8 +1604,8 @@ PalettePtr PaletteCreator::newTextPalette(bool defaultPalette)
         sp->appendElement(meaNum, QT_TRANSLATE_NOOP("palette", "Measure number"))->setElementTranslated(true);
 
         static const std::vector<PlayTechAnnotationInfo> playTechAnnotationsMaster = {
-            { QT_TRANSLATE_NOOP("palette", "détaché"),   PlayingTechniqueType::Detache },
-            { QT_TRANSLATE_NOOP("palette", "martelé"),   PlayingTechniqueType::Martele },
+            { QT_TRANSLATE_NOOP("palette", "dÃ©tachÃ©"),   PlayingTechniqueType::Detache },
```

Comment:
More uncrustify rubbish

Reply:
I see this had not been fixed before the PR was merged. I'll fix it now, but it might be good to double-check that that doesn't mean that any other fixes had not been pushed either.

Reply:
I probably did fix it and then Uncrustify messed it up again as it runs on every save. Do we know if that's an Uncrustify or a Qt Creator bug? I may need to just disable Uncrustify altogether, it's way to easy to accidentally push garbage